rtuuy/supporter-readme.yaml

```diff
@@ -68,4 +67,4 @@
       description: "Get pinged for updates and announcements about the supporter server."
 
 - type: text
-  text: <a:ratspin:924749326310309959> <:ratmaidplush:1048622137050669056>
+  text: https://cdn.discordapp.com/attachments/1074017809316728852/1074044703860871268/Rat_supporter_mods__config.zip
```

Comment:
My reason for not doing this is because RAT requested moderators be able to change the pack but I guess it's fine for now

Reply:
Oh I see. I thought that it didn't make any sense because on top it said "mod pack below" yet the pack was nowhere to be seen.

Reply:
As I was hoping not for but expecting anyway, welcome-channels removes non-bot messages from "welcome channels"